README.md

```diff
@@ -10,30 +10,22 @@ status: draft
 
 # Introduction
 
-In this tutorial, we are going to build a simple web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com).
+In this tutorial, we are going to build a web application using [AdonisJS](https://adonisjs.com/) and integrate it with [Materialize](https://materialize.com) to .
 
-# Prerequisits
+# Prerequisites
 
-You would need to have the following things installed before you begin:
+You need to have the following things installed before getting started:
 
-* [Install Docker and Docker Compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04)
-* [Install Node.js](https://devdojo.com/ruanbekker/install-nodejs-on-linux-using-nvm)
+* [Docker and Docker Compose](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-compose-on-ubuntu-20-04)
+* [Node.js](https://devdojo.com/ruanbekker/install-nodejs-on-linux-using-nvm)
 
 # What is Materialize
 
-**[Materialize](https://materialize.com) is a streaming database** that takes sources of data coming from sources like Kafka, PostgreSQL, S3 buckets, and more and allows you to create views that aggregate data on your event stream.
+**[Materialize](https://materialize.com) is a streaming database** that takes data coming from sources like Kafka, PostgreSQL, S3 buckets, and more and allows you to effectively transform it in real-time using SQL.
 
-The magic is that the views are translated under the hood to dataflows and it allows Materialize to maintain the views incrementally in real-time. 
```

Comment:
Same comment as last time! Going down the dataflow route is possibly the hardest way to demonstrate Materialize's value to new users.